### PR TITLE
Updated the library and demo application.

### LINKED ISSUE #1: Obsolete build syntax in build.gradle file.
Hi!

The _build.gradle_ file contains some obsolete syntax for the compilation of the different parts of the Android application. If you are willing, I would like to love to submit a PR to update them to latest syntax versions.

Thanks.

Regards,
Dheeraj Nair

### PR DESCRIPTION
**Pull request by Dheeraj1998**
Hey there! This PR has been made to resolve #1. The changes made include:

- Updated the build version for the library file in the highlight folder.
- The demo application has been updated according for the latest Gradle version.
- New screenshots has been added and ReadMe has been updated.

You can see the changes made live at: https://github.com/Dheeraj1998/Highlight.

Do let me know if any changes are to be made 😸

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ highlightManager.addView(R.id.some_widget).setTitle(R.string.title)
 ~~~
 
 Refer to the
-[demo app](https://github.com/apuder/Highlight/blob/master/demo/src/main/java/org/puder/highlight/demo/MainActivity.java)
+[demo app](https://github.com/Dheeraj1998/Highlight/blob/master/demo/app/src/main/java/org/puder/highlight/demo/MainActivity.java)
```
Comment (apuder) [inline README.md]: Could you please make that URL point to my repo?

**Review by apuder — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Dheeraj1998) [reply README.md]: Yup, URL has been modified.

**Comment by Dheeraj1998** [comment]
The requested changes have been done!

**Comment by apuder** [comment]
Thanks for your PR!

**Pull request merged**